.gitmodules

```diff
@@ -2,3 +2,6 @@
 [submodule "src/SkyApm.Transport.Grpc.Protocol/protocol"]
```

Comment:
Old submodule should be removed.

Reply:
for backward compatiblility, currently is supporting both v8.x and bellow version protocol. switching by configuration.

Reply:
So, you have config to activate v2 protocol?

Reply:
The sdk contains both v2 and v3 protocol code, the users do the choice by editing the `Transport.ProtocolVersion` and `HeaderVersions` sections in the `skyapm.json` configuration file.

Reply:
Good to know.